addons/payment/views/payment_form_templates.xml

```diff
@@ -412,7 +412,7 @@
             <!-- === Availability report button (for debug) === -->
             <t t-call="payment.availability_report_button"/>
             <br/>
-            If you believe that it is an error, please contact the website administrator.
+            If you believe that it is an error, please contact the administrator.
```

Comment:
I don't see anything in specs to change this :worried: 
Can you tell person who ask for this change to add it into specs or add this into specs by their tri-gram

Reply:
Yes, I had updated the spec.

Reply:
I think here DALA saying change error message for admin user as `please contact the administrator` make no sense to admin user :thinking: